css/css-basics_pokemon-page/css/styles.css

```diff
@@ -45,7 +51,15 @@
 */
 
 section {
-  /* your solution */
+  background-color: white;
+  margin: 10px;
+  padding: 20px;
+  border-style: solid;
+  border-color: orange;
+  border-width: 5px;
+}
+section:nth-of-type(2) {
```

Comment:
Did this work for you? Didn't for me, you might check again and maybe find a different solution if it didn't like in my case. (If it does - nice solution!)

Reply:
actually yes... It's working for me.
I can show you if you want 😁